services/soap/src/main/java/org/mule/services/soap/impl/xml/util/XMLUtils.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.services.soap.impl.xml.util;
+
+import org.dom4j.io.DOMReader;
+import org.w3c.dom.Document;
+
+/**
+ * General utility methods for working with XML.
+ */
+public class XMLUtils extends org.mule.runtime.core.util.XMLUtils {
```

Comment:
in this case, this is just the code needed by the soap service and the wsc. I'll create a ticket so this can be tracked separately from those artifacts (which are related.)